datarepo-clienttests/src/main/java/runner/TestScriptResult.java

```diff
@@ -0,0 +1,77 @@
+package runner;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.util.List;
+import runner.config.TestScriptSpecification;
+
+public class TestScriptResult {
+  public List<UserJourneyResult> userJourneyResults;
+  public TestScriptResultSummary summary;
+
+  /**
+   * Summary statistics are pulled out into a separate inner class for easier summary reporting.
+   * This class does not include a reference to the full TestScriptSpecification or the list of
+   * UserJourneyResults.
+   */
+  @SuppressFBWarnings(
+      value = "URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD",
+      justification = "This POJO class is used for easy serialization to JSON using Jackson.")
```

Comment:
Is this needed? It looks like this class is referenced elsewhere.

Reply:
Yes, it's needed for spotbugs to pass. Several of the properties are read only by the Jackson JSON-writer, and so spotbugs thinks they're never used. This annotation just overrides it for all fields in the class.